src/app/app-modules/core/components/app-header/app-header.component.html

```
@@ -1,14 +1,15 @@
-<header class="app-header">
-  <a class="app-header-brand" aria-label="Piramal Swasthya">
-    <img src="assets/images/Swasthya.png" alt="Piramal Swasthya" />
+<header
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)